sdk/schemaregistry/Azure.Data.SchemaRegistry/README.md

```diff
@@ -0,0 +1,157 @@
+
+```C# Snippet:CreateSchemaRegistryClient
+string endpoint = "<event_hubs_namespace_hostname>";
+var credentials = new ClientSecretCredential(
+    "<tenant_id>",
+    "<client_id>",
+    "<client_secret>"
+);
+var client = new SchemaRegistryClient(endpoint, credentials);
+```
+
+## Key concepts
+
+### Schemas
+
+A schema has 6 components:
+- Group Name
+- Schema Name
+- Schema ID
+- Serialization Type
+- Schema Content
+- Schema Version
+
+These components play different roles. Some are used as input into the operations and some are outputs. Currently, [SchemaProperties][schema_properties] only exposes those properties that are potential outputs that are used in SchemaRegistry operations. Those exposed properties are `Content` and `Id`.
+
+## Examples
+
+The following shows examples of what is available through the SchemaRegistryClient. There are both sync and async methods available for these client operations.
+
+* [Register a schema](#register-a-schema)
+* [Retrieve a schema ID](#retrieve-a-schema-id)
+* [Retrieve a schema](#retrieve-a-schema)
+
+### Register a schema
+
+Register a schema to be stored in the Azure Schema Registry.
+
+```C# Snippet:RegisterSchema
+            string schemaName = "<schema_name>";
+            string groupName = "<schema_group_name>";
+            SerializationType schemaType = SerializationType.Avro;
+            // Example schema's content
+            string schemaContent = @"
+{
+   ""type"" : ""record"",
+    ""namespace"" : ""TestSchema"",
+    ""name"" : ""Employee"",
+    ""fields"" : [
+    { ""name"" : ""Name"" , ""type"" : ""string"" },
+    { ""name"" : ""Age"", ""type"" : ""int"" }
+    ]
+}";
+
+            Response<SchemaProperties> schemaProperties = client.RegisterSchema(groupName, schemaName, schemaType, schemaContent);
+```
+
+### Retrieve a schema ID
+
+Retrieve a previously registered schema ID from the Azure Schema Registry.
+
+```C# Snippet:RetrieveSchemaId
+            string schemaName = "<schema_name>";
+            string groupName = "<schema_group_name>";
+            SerializationType schemaType = SerializationType.Avro;
+            // Example schema's content
+            string schemaContent = @"
+{
+   ""type"" : ""record"",
+    ""namespace"" : ""TestSchema"",
+    ""name"" : ""Employee"",
+    ""fields"" : [
+    { ""name"" : ""Name"" , ""type"" : ""string"" },
+    { ""name"" : ""Age"", ""type"" : ""int"" }
+    ]
+}";
+
+            Response<SchemaProperties> schemaProperties = client.GetSchemaId(groupName, schemaName, schemaType, schemaContent);
+            string schemaId = schemaProperties.Value.Id;
+```
+
+### Retrieve a schema
+
+Retrieve a previously registered schema's content from the Azure Schema Registry.
+
+```C# Snippet:RetrieveSchema
+string schemaId = "<schema_id>";
+
+Response<SchemaProperties> schemaProperties = client.GetSchema(schemaId);
+string schemaContent = schemaProperties.Value.Content;
+```
+
```

Comment:
Should there be a troubleshooting section?

Reply:
I've removed it as I have no troubleshooting information right now. I originally had the section with a message like "This information will be added in the future" or something to that affect, but it seemed stupid. It is a section I'll add back once there is information to provide. Added issue for it: https://github.com/Azure/azure-sdk-for-net/issues/14934